agent/src/main/scala/za/co/absa/atum/agent/AtumContext.scala

```diff
@@ -120,6 +120,17 @@ class AtumContext private[agent] (
     this
   }
 
+  /**
+   * This method creates `Metadata` in the agentService.
+   *
+   * @return AtumContext
+   */
+  def AdditionalData(): AtumContext = {
```

Comment:
functions and methods in Scala start with lowercase: https://docs.scala-lang.org/style/naming-conventions.html

Reply:
on few other places as well